ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -380,12 +383,20 @@ struct MSFListCellView: View, ConfigurableTokenizedControl {
             let horizontalCellPadding: CGFloat = tokens.horizontalCellPadding
             let leadingViewAreaSize: CGFloat = tokens.leadingViewAreaSize
 
-            Button(action: state.onTapAction ?? {
+            Button(action: {
                 if hasChildren {
                     withAnimation {
                         state.isExpanded.toggle()
                     }
                 }
+
+                var handledSelection = false
+                if let selectionAction = state.selectionAction {
+                    handledSelection = selectionAction(state)
+                }
+                if !handledSelection, let onTapAction = state.onTapAction {
```

Comment:
If both should be possible for a single cell (which seems like it should be the case, since that's how things like `UITableViewCell` work), then we won't need a boolean return type.

Reply:
When I was talking to @rdeassis about it, it sounded like having both would be redundant. I can change it to allow for both, though, if that's what we want.